app/templates/email/bug-report.ts

```diff
@@ -0,0 +1,140 @@
+export const bugReportTemplates = {
+  en: `
+Describe the bug
+A clear and concise description of what the bug is. If it seems connected to some data problem (missing values, wrong parsing), please first check the cube in Cube Validator to see if everything is fine there.
```

Comment:
Nit: I know it comes probably from our GitHub error template, but I'd add a link here (https://cube-validator.lindas.admin.ch/select), I guess most of the users don't know what a Cube Validator is (also in other languages).

Reply:
Ah, in fact we have a link in GitHub template, I'd also add it here :)